docs/source/fx.rst

```diff
@@ -7,250 +7,181 @@ Overview
-it may be easier to examine modules and parameters using ``to_folder``.
-
-::
-
-    m = symbolic_trace(M())
-    m.to_folder("foo", "Bar")
-    from foo import Bar
-    y = Bar()
-
-After running the above example, we can then look at the code within
-``foo/module.py`` and modify it as desired (e.g. adding ``print``
-statements or using ``pdb``) to debug the generated code.
-
-Debugging the Transformation
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Now that we've identified that a transformation is creating incorrect
-code, it's time to debug the transformation itself. First, we'll check
-the :ref:`Limitations of Symbolic Tracing` section in the documentation.
-Once we verify that ``symbolic_trace`` is working as expected, the goal
-becomes figuring out what went wrong during our ``GraphModule``
-transformation. There may be a quick answer in
-:ref:`Writing Transformations`, but, if not, there are several ways to
-examine our traced module:
-
-::
-
-    # Sample Module
+    # Sample module
     class M(torch.nn.Module):
         def forward(self, x, y):
-            return x + y
-
-    # Create an instance of `M`
-    m = M()
-
-    # Symbolically trace an instance of `M` (returns a GraphModule). In
-    # this example, we'll only be discussing how to inspect a
-    # GraphModule, so we aren't showing any sample transforms for the
-    # sake of brevity.
-    traced = symbolic_trace(m)
-
-    # Print the code produced by tracing the module. The generated `forward`
-    # function is:
-    """
-    def forward(self, x, y):
-        add_1 = x + y;  x = y = None
-        return add_1
-    """
-    print(traced)
-
-    # Print the internal Graph. This representation returns:
-    """
-    graph(x, y):
-        %add_1 : [#users=1] = call_function[target=<built-in function add>](args = (%x, %y), kwargs = {})
-        return add_1
-    """
-    print(traced.graph)
-
-    # Print a tabular representation of the internal Graph. This gives us:
-    """
-    opcode         name    target                   args      kwargs
-    -------------  ------  -----------------------  --------  --------
-    placeholder    x       x                        ()        {}
-    placeholder    y       y                        ()        {}
-    call_function  add_1   <built-in function add>  (x, y)    {}
-    """
-    traced.graph.print_tabular()
-
-Using the utility functions above, we can compare our traced Module
-before and after we've apply our transformations. Sometimes, a
-simple visual comparison is enough to trace down a bug. If it's still
-not clear what's going wrong, a debugger like ``pdb`` can be a good
-next step.
-
-Going off of the example above, consider the following code:
+            return torch.add(x, y)
+
+    def transform(m: nn.Module) -> nn.Module:
+        fx_model: GraphModule = fx.symbolic_trace(m)
+        # FX represents its graph as an ordered list of nodes, so we can iterate through them.
```

Comment:
Can we wrap these lines to 60 chars or so? The docs page has table of contents bars on the left and right and the code is harder to read when it's wrapping around